cmd/run.go

```diff
@@ -22,16 +22,25 @@ var runCmd = &cobra.Command{
 			return err
 		}
 
+		force_jitsi, err := cmd.Flags().GetBool("jitsi")
+		if err != nil {
+			return err
+		}
+
 		manual := false
 		if url == "" {
 			manual = true
 		}
 
-		// TODO: When additional video conference platforms are added, make this
-		// url evaluation more sophisticated.
+		var scrape_func = scrape.GetParticipantsZoom
 		if !manual {
-			if !strings.Contains(url, "zoom") {
-				return fmt.Errorf("provided url does not contain 'zoom'")
+			// Checking optional force_jitsi flag first
```

Comment:
I recommend converting this into a switch case now that it has this many branches. Checkout https://goplay.space/#vT6KcxJFOgN.